common/src/main/scala/io/phdata/services/EmbeddedImageEmail.scala

```diff
@@ -0,0 +1,26 @@
+package io.phdata.services
+
+import courier.Multipart
+import javax.mail.internet.MimeBodyPart
+
+class EmbeddedImageEmail {
```

Comment:
I don't see a reason this class has to exist, can everything go in the object instead?